server/build.gradle

```diff
@@ -190,6 +190,7 @@ dependencies {
   compileOnly project(':app-server')
   compileOnly group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.47'
   compileOnly group: 'org.eclipse.jetty.websocket', name: 'websocket-server', version: versions.jetty
+  compile group: 'org.eclipse.jetty.websocket', name: 'javax-websocket-server-impl', version: versions.jetty
```

Comment:
@ketan nevermind, I just tried: gradle will not compile without these declared in `server`.